examples/dns-interceptor/src/pipeline.rs

```diff
@@ -67,11 +67,16 @@ impl route_rs_runtime::pipeline::Runner for Pipeline {
             .build_link();
         all_runnables.append(&mut runnables_6);
 
-        tokio::run(lazy(move || {
-            for r in all_runnables {
-                tokio::spawn(r);
+        let mut rt = runtime::Builder::new().enable_all().build().unwrap();
+
+        rt.block_on(async move {
+            let mut handles = vec![];
```

Comment:
Construction via `map`? _#unnecessarilyfunctional_

Reply:
:heck:

Reply:
gong to stick with simple for now here, when we revamp the new public facing running harness then I for one welcome our functional overlords.